java/memory/src/main/java/io/netty/buffer/NettyArrowBuf.java

```diff
@@ -156,7 +156,7 @@ public ByteBuf touch(Object hint) {
 
   @Override
   public int capacity() {
-    return arrowBuf.capacity();
+    return (int)Math.min(Integer.MAX_VALUE, arrowBuf.capacity());
```

Comment:
A space should be inserted after "(int)"?

Reply:
done.